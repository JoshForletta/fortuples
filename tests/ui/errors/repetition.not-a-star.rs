use fortuples::fortuples;

struct Container<T>(T);

fortuples! {
    impl Container<#Tuple> {} #(),$
}

fn main() {}
