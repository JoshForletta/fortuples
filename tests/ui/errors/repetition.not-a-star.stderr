error: `*` is expected
 --> tests/ui/errors/repetition.not-a-star.rs:6:35
  |
6 |     impl Container<#Tuple> {} #(),$
  |                                   ^
